data/diff-tests/outputs/trailing-blank-line.txt

```diff
@@ -0,0 +1,4 @@
+TEST
+@@ -1,2 +1,1 @@
+  data Test = Test
```

Comment:
I think that the typical behavior will be that Ormolu will add the missing trailing blank line, not remove it.